git_machete/exceptions.py

```diff
@@ -4,7 +4,7 @@
 
 
 class MacheteException(Exception):
-    def __init__(self, msg: str, apply_fmt: bool = True, verbose: bool = True) -> None:
+    def __init__(self, msg: str, apply_fmt: bool = True, verbose: bool = False) -> None:
         self.parameter = utils.fmt(msg) if apply_fmt else msg
         if not verbose:
```

Comment:
Nit: if this `if` stays, pls flip the condition to remove the negation

Reply:
outdated, if removed.